.github/workflows/release-drafter.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
```

Comment:
I don't know what this action does exactly (I don't think I've ever run it), but it might need write permissions too

Reply:
same as above.

Reply:
It seems OK https://github.com/danyeaw/release-drafter/blob/ca78f853c4403f81294579c6fbff6acd2632795e/README.md